lib/facter/monit_version.rb

```diff
@@ -0,0 +1,14 @@
+# A facter fact to determine the installed version of monit.
+
+module Facter::Util::MonitVersion
+  class << self
+    def get_monit_version
+      monit_version = Facter::Util::Resolution.exec('monit -V 2>&1')
```

Comment:
@petems I ran this on a system without monit and it does not cause facter to throw any messages or report anything to STDERR

CC @Phil-Friderici who worked on this piece of code